elasticreindexer/elastic/elasticClient.go

```diff
@@ -31,9 +30,16 @@ type esClient struct {
 // NewElasticClient will create a new instance of an esClient
 func NewElasticClient(cfg config.ElasticInstanceConfig) (*esClient, error) {
 	elasticClient, err := elasticsearch.NewClient(elasticsearch.Config{
-		Addresses: []string{cfg.URL},
-		Username:  cfg.Username,
-		Password:  cfg.Password,
+		Addresses:     []string{cfg.URL},
+		Username:      cfg.Username,
+		Password:      cfg.Password,
+		RetryOnStatus: []int{429, 502, 503, 504},
```

Comment:
Could be this extracted into a variable with an explicit name?

Reply:
done